grpc-test/src/main/java/com/hazelcast/jet/tests/grpc/GrpcTest.java

```diff
@@ -0,0 +1,141 @@
+/*
+ * Copyright (c) 2008-2018, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.jet.tests.grpc;
+
+import com.hazelcast.collection.IList;
+import com.hazelcast.jet.JetInstance;
+import com.hazelcast.jet.config.JobConfig;
+import com.hazelcast.jet.grpc.GrpcService;
+import com.hazelcast.jet.pipeline.Pipeline;
+import com.hazelcast.jet.pipeline.ServiceFactory;
+import com.hazelcast.jet.pipeline.Sinks;
+import com.hazelcast.jet.pipeline.Sources;
+import com.hazelcast.jet.tests.common.AbstractSoakTest;
+import com.hazelcast.jet.tests.grpc.greeter.GreeterGrpc;
+import com.hazelcast.jet.tests.grpc.greeter.GreeterOuterClass.HelloReply;
+import com.hazelcast.jet.tests.grpc.greeter.GreeterOuterClass.HelloRequest;
+import com.hazelcast.map.IMap;
+import io.grpc.ManagedChannelBuilder;
+import io.grpc.Server;
+
+import java.util.Map;
+
+import static com.hazelcast.jet.grpc.GrpcServices.bidirectionalStreamingService;
+import static com.hazelcast.jet.grpc.GrpcServices.unaryService;
+import static com.hazelcast.jet.impl.util.Util.uncheckCall;
+import static com.hazelcast.jet.tests.common.Util.sleepSeconds;
+import static com.hazelcast.query.Predicates.alwaysTrue;
+import static java.util.stream.Collectors.toList;
+import static java.util.stream.IntStream.range;
+
+public class GrpcTest extends AbstractSoakTest {
+
+    private static final String SOURCE_MAP_NAME = "grpcSourceMap";
+    private static final String SINK_LIST_NAME = "grpcSinkList";
+
+    private static final int DEFAULT_ITEM_COUNT = 10_000;
```

Comment:
Not exactly related to gRPC, but this seems rather low for a soak test which is going to run on a server with high core count. It will not fill all the queues etc..

Reply:
we have other tests running on the cluster too which stresses the cluster already. We can increase it to 100K maybe

Reply:
I've increased it to 100K on the ansible side and kept the default as is for local testing

https://github.com/hazelcast/hazelcast-jet-ansible/pull/55